### ISSUE TITLE
[Accessibility] A11y_Manage.IIS.Net_Web Server_Authentication_Keyboard: Focus is not visible on some of the controls in all the pages

### ISSUE THREAD

**Issue by InduPriya1805** [Microsoft/IIS.WebManager#89]
GitHubTags: #A11y_Manage.IIS.Net; #A11yMAS; #A11yTCS; #C+E-H; #DesktopWeb; #Win10; #.NETWebsites; #Edge; #Chrome; #Keyboard; #MAS2.4.7

**Environment Details:**
URL: https://manage.iis.net/webserver/authentication
Browser Details: Microsoft Edge 42.17134.1.0, Google Chrome 68.0.3440.106
Keyboard
MAS ID Violated: MAS 2.4.7

**Repro Steps:**
1. Download and install "Microsoft IIS Administration" application.
2. Enable Internet information Services.
3. Open "Microsoft IIS" application using the URL: https://manage.iis.net/
4. Tab till Web Server and press enter
5. Tab till Authentication and press enter
6. Validate focus is visible on all the controls or not

**Actual Result:**
Focus is not visible on some of the controls in the page

**Expected Result:**
Focus should visible on all the controls in the page

**User Impact:**
Users with disabilities finds its difficult in working through the application if focus is not properly visible on all the controls

**Note:**
Similar issue is also found in Web Server- Authorization
                                               Web Server- Authorization-Create rule
                                               Web Server- IP Restrictions

Attachments:
![bug89_web server_authentication_focus visible1](https://user-images.githubusercontent.com/42638450/44643723-323df800-a9ef-11e8-9e47-b1d7423d1b4a.png)
![bug89_web server_authentication_focus visible2](https://user-images.githubusercontent.com/42638450/44643724-32d68e80-a9ef-11e8-9f5d-2eb436c9aad4.png)
![bug89_web server_authentication_focus visible3](https://user-images.githubusercontent.com/42638450/44643726-32d68e80-a9ef-11e8-82ae-f1207be38312.png)
![bug89_web server_authentication_focus visible4](https://user-images.githubusercontent.com/42638450/44643727-32d68e80-a9ef-11e8-9a3e-68bcb6f9e267.png)
![bug89_web server_authorization_create rule_focus visible5](https://user-images.githubusercontent.com/42638450/44776587-0e68e680-ab96-11e8-9415-3d0553b05ace.png)
![bug89_web server_authorization_focus visible6](https://user-images.githubusercontent.com/42638450/44776588-0e68e680-ab96-11e8-9acd-d0fffa034850.png)
![bug89_web server_ip restrictions_focus visible7](https://user-images.githubusercontent.com/42638450/44776590-0e68e680-ab96-11e8-8de6-5d4089a2c2b4.png)
![bug89_web server_ip restrictions_focus visible8](https://user-images.githubusercontent.com/42638450/44776591-0f017d00-ab96-11e8-89f0-357e937d6b91.png)

**Comment by jhkimnew** [comment]
Fixed

**Comment by Roopesh-T** [comment]
**GitTags**: #A11ySev1;

**Comment by Yashi1201** [comment]
#Closed;
Manage.iis.net is deprecated. The "Manage IIS" button on the www.iis.net home page will be removed.
